share/Makefile

```diff
@@ -4,12 +4,8 @@ MOFILES := $(POFILES:.po=.mo)
 MSGFMT  = msgfmt --check
```

Comment:
I think we should switch to Perl msgfmt instead of binary.

Reply:
Done